Makefile

```diff
@@ -409,7 +409,7 @@ clean-helm:
 install-kind: "$(KIND)"
 
 "$(GOBIN)/crane":
-	go install github.com/google/go-containerregistry/cmd/crane@$(CRANE_VERSION)
+	CGO_ENABLED=0 go install github.com/google/go-containerregistry/cmd/crane@$(CRANE_VERSION)
```

Comment:
why is CGO_ENABLED=0 needed?

Reply:
The e2e container doesn't have glibc libraries installed, which causes a linker error when trying to run the crane binary from inside the container (e.g. for the tests).
```
crane: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by crane)
crane: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by crane)
```

Reply:
Should we set CGO_ENABLED=0 in the Dockerfile?

Reply:
It is set in the dockerfile, the issue here is that `crane` is built outside of the e2e container. So for localhost it would be built on the host machine and in prow it would be built inside the kubekins container.

Reply:
Ahh, I see. Then maybe this should go in another PR to fix it with a comment that says why crane and CGO_ENABLED=0 aren't just in a container.

Reply:
https://github.com/GoogleContainerTools/kpt-config-sync/pull/1107